src/components/Reports/SalaryCalculator/PersonalInformationSection/PersonalInformationSection.tsx

```diff
@@ -0,0 +1,84 @@
+import Link from 'next/link';
+import React from 'react';
+import {
+  CardContent,
+  CardHeader,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+import { StepCard, StepTableHead } from '../Shared/StepCard';
+import { usePersonalInformation } from './usePersonalInformation';
+
+export const PersonalInformationSection: React.FC = () => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const {
+    spouse,
+    selfLocation,
+    spouseLocation,
+    selfTenure,
+    spouseTenure,
+    selfAge,
+    spouseAge,
+    selfChildren,
+    spouseChildren,
+  } = usePersonalInformation();
+
+  return (
+    <StepCard>
+      <CardHeader
+        title={t('Personal Information')}
+        data-testid="personal-information-header"
+      />
+      <CardContent>
```

Comment:
It seems like we'd always want typography inside `StepCard` --> `CardContent` to line up with the header text. Can we add the horizontal margin to the '.MuiCardContent-root .MuiTypography-root' rule in `StepCard`?